src/Arcus.Template.WebApi/Startup.cs

```diff
@@ -11,6 +12,9 @@ public class Startup
         public void ConfigureServices(IServiceCollection services)
         {
             services.AddMvc();
+
+            services.AddHealthChecks()
+                    .AddCheck("self", () => HealthCheckResult.Healthy());
```

Comment:
What happens if we leave this one out?

Reply:
I don't think you need the 'self' check.